services/management/management.go

```diff
@@ -0,0 +1,90 @@
+package management
+
+import (
+	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/api/inventorypb"
+	"github.com/percona/pmm/api/managementpb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+func nodeID(tx *reform.TX, nodeID, nodeName string, addNodeParams *managementpb.AddNodeParams, address string) (string, error) {
+	if err := validateNodeParamsOneOf(nodeID, nodeName, addNodeParams); err != nil {
+		return "", err
+	}
+	switch {
+	case nodeID != "":
+		return nodeID, nil
+	case nodeName != "":
+		node, err := models.FindNodeByName(tx.Querier, nodeName)
+		if err != nil {
+			return "", err
+		}
+		return node.NodeID, nil
+	case addNodeParams != nil:
+		node, err := addNode(tx, addNodeParams, address)
+		if err != nil {
+			return "", err
+		}
+		nodeID = node.NodeID
```

Comment:
```suggestion
		return node.NodeID
```

That way all case branches will have `return`, and no `return` is necessary after the `switch`.